docs/section12/Rule12-1.md

```diff
@@ -1,36 +1,37 @@
-
-  - For each thermal_block in building segment: ```thermal_block_user in building_segment_user.thermal_blocks:```
-
-  - For each zone in thermal block: ```zone_user in thermal_block_user.zones:```
-
-  - For each space in zone, add to the total number of spaces in the User model: ```for space_user in zone_user.spaces: num_of_spaces_user += 1```  
-
-- Get the total number of spaces in the building segment in the Baseline model: ```for building_segment_baseline in B_RMR.building.building_segments:```  
-
-  - For each thermal_block in building segment: ```thermal_block_baseline in building_segment.thermal_blocks:```
-
-  - For each zone in thermal block: ```zone_baseline in thermal_block_baseline.zones:```
-
-  - For each space in zone, add to the total number of spaces in the Baseline model: ```for space_baseline in zone_baseline.spaces: num_of_spaces_baseline += 1```  
-
-**Rule Assertion:** ```num_of_spaces_user == num_of_spaces_baseline```  
-
-**[Back](../_toc.md)**
+**Evaluation Context:** Each RPD  
+
+**Applicability Checks:**  
+N/A
+
+**Function Call:**  
+match_data_element
+
+## Rule Logic:
+- Store a list of any miscellaneous equipment power where proposed is greater than baseline: `unexpected_misc_equipment_power = []`
+- Store a list of any miscellaneous equipment power where proposed is less than baseline: `reduced_misc_equipment_power = []`
+- For each miscellaneous equipment load in the baseline RMD: `for misc_equipment_b in B_RMD...miscellaneous_equipment:`
+  - Get the corresponding miscellaneous equipment load in the proposed RMD: `misc_equipment_p = match_data_element(P_RMD, MiscellaneousEquipment, misc_equipment_b.id)`
+  - Get the miscellaneous equipment power in the proposed RMD: `misc_equipment_power_p = misc_equipment_p.power`
+  - Get the miscellaneous equipment power in the baseline RMD: `misc_equipment_power_b = misc_equipment_b.power`
+  - If the miscellaneous equipment power in the baseline RMD is greater than the miscellaneous equipment power in the proposed RMD: `if misc_equipment_power_b > misc_equipment_power_p:`
+    - Append the miscellaneous equipment info for reporting: `reduced_misc_equipment_power.append({"id": misc_equipment_b.id, "baseline_power": misc_equipment_power_b, "proposed_power": misc_equipment_power_p})`
+  - If the miscellaneous equipment power in the baseline RMD is less than the miscellaneous equipment power in the proposed RMD: `elif misc_equipment_power_b < misc_equipment_power_p:`
+    - Append the miscellaneous equipment info for reporting: `unexpected_misc_equipment_power.append({"id": misc_equipment_b.id, "baseline_power": misc_equipment_power_b, "proposed_power": misc_equipment_power_p})`
+**Rule Assertion:**  
+  - Case 1: If all equipment power was modeled identically: PASS `if len(unexpected_misc_equipment_power) == 0 and len(reduced_misc_equipment_power) == 0: PASS`
```

Comment:
My understanding is Rule 1-7 will check and highlight these outcome so this rule essentially, provides more detail messages about the failures?

Reply:
MiscellaneousEquipment.power is set to FALSE for the AppG B_RMD Equals P_RMD column in the extra schema so 1-6 will not check this (1-7 is for proposed vs. user). False is appropriate for 1-6 and we check in this rule instead because when the project is for a beyond-code program the proposed is allowed to be less than the baseline, but we are unable to tell if the reduction was correctly applied for the beyond code program so outcome is UNDETERMINED. 

If proposed is greater than the baseline, FAIL. Also, as Christina pointed out, `if RPD.compliance_path == CODE_COMPLIANT`, any difference between proposed and baseline is a FAIL.